themes/conventional-branch/static/css/scss/abstracts/_variables.scss

```
@@ -1,9 +1,14 @@
 $color-primary: #6699CC;
 $color-primary-light: #6699CC;
+$color-primary-deep: #3B6BB5;
 $color-neutral-light: #FFF;
 $color-neutral-dark: #000;
 
-$color-primary-heading: #7fa5ec;
+$color-primary-heading: #2B5EA7;
```

Stylelint (17.13.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)